---
title: "Research Contributions"
weight: 30
---


### Mechanisms of microbial pathogen resistance (2012-2015)
I worked on two microbiology research during my undergraduate training at the University of North Georgia, Dahlonega Campus. 

__Antibiotic Resistance mechanisms of the gut microbe Bacteroides spp. (2012-2014)__

I studied a putative tripartite efflux pump with high homology to the well studied E. coli _AcrE-AcrF-TolC_ pump by using perform genetic complementation and drug susceptibility assays. This work was performed under the mentorship of Paul Johnson, PhD. 

[Abstract](https://drive.google.com/file/d/1wYKErS21Zy41gvfiAymo-kUuG8xv0kEG/view?usp=sharing)

__Abundance and Diversity of Halophiles on the salt pans of Sapelo Island (2013)__

Two of my classmates and I traveled to Sapelo Island, GA to explore the halophiles on the salt pans of sapelo island. We found that higher salinity salt pans are usually further away from freshwater and that the higher salinity salt pans have both high abundance and diversity of halophiles! This work was performed under the co-mentorship of Nancy Dalman, PhD and Paul Johnson, PhD. Links: 

[Abstract](https://drive.google.com/file/d/1mu-nhsYX59DDBb7Nm1iomt6_RVUZ2GIu/view?usp=sharing), [Local Poster Presentation](https://drive.google.com/file/d/1CA_4-51N78UhdDOolabT8XC8DAdkTt0T/view?usp=sharing), see my [blog post](https://www.pmhtran.com/blog/sapelo/) about my experience on Sapelo Island!

__Post-transcriptional regulation of csrA through the FliW chaperone protein in Campylobacter jejuni (2014)__

I rotated in the laboratory of Stuart Thompson, PhD. We studied with role of the chaperone protein FliW in regulating the transcription factor csrA in Campylobacter jejuni. This transcription facter plays a critical role in biofilm formation in Campylobacter spp.

[Manuscript 1](https://www.frontiersin.org/articles/10.3389/fmicb.2020.531596/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Microbiology&id=531596)


### Biomarkers of cancer classication and prognosis (2014-2020)
During my early years in medical and graduate school, I applied machine learning methods to genomics data to help with classification and predicting poor prognosis in various cancers. I formulated the project and conducted all of the brain cancer project. For the gynecologic cancer projects, I was involved in patient consent, specimen collection, sample processing, data curation, data analysis, and manuscript writing.

__Transcriptomic Classification of Gliomas__

This project is a major part of my thesis. I used publicly available glioma transcriptomic data and applied a custom unsupervised classification algorithm using a combination of [UMAP](https://umap-learn.readthedocs.io/en/latest/) and [density-based clustering](https://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.121.9220). I identified four clusters highly similar to previously published IDH-codel groups and methylomic groups. The new clusters may better reflect patient survival compared to previously reported glioma groups. I validated these findings in a local retrospective cohort.

[Manuscript 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7692499/),
[AACR 2019 Poster Presentation](https://drive.google.com/file/d/1wXC0KHW8G7mtk3vVksCtgmrueqz6VTCg/view?usp=sharing), [news article](https://mindzilla.com/newsroom/science/genomics/44048/20210119-new-approach-emerges-to-better-classify-treat-brain-tumours/), and [local news channel feature](https://www.wfxg.com/story/43206589/doctors-at-au-create-new-approach-to-better-classify-and-treat-brain-tumors)

[Manuscript 2](https://www.mdpi.com/2072-6694/13/3/439) and 
[Code](https://github.com/pmtran5884/Brain-paper-2)

__Serum markers of Poor Prognosis in a Peruvian Cervical Cancer Cohort__

In this project, I helped analyze data from the [Cervicusco](http://cervicusco.org/en/about_cc/compassion-in-care/) project head by [Daron Ferris, MD](https://www.augusta.edu/cancer/research/labs/inactive-daron-ferris.php). We wanted to identify serum markers of poor prognosis that could be used in the low resource settings in Peru.

The first manuscript identifies anti-carbohydrate antibodies (ACAs) using a [custom array](https://www.nature.com/articles/s41467-017-02747-y) developed by Sharad Purohit, PhD and my mentor, Jin-Xiong She, PhD. [Manuscript](https://www.gynecologiconcology-online.net/article/S0090-8258(20)30023-8/fulltext)

The second manuscript identifies serum proteins detected on a muliplex immunoassay associated with poor prognosis.
[Manuscript](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7601905/)


### Genetic and Environmental Determinants of Islet Autoimmunity, Type 1 Diabetes, and Diabetic Complications (2018-2021)
During my later years in graduate school, I applied machine learning methods to genetic, clinical, and serum protein data to help with predicting islet autoimmunity and diabetic complications.

__Polygenic Risk Score for Islet Autoimmunity__

I was fortunate to receive [funding](https://projectreporter.nih.gov/project_info_description.cfm?aid=9976989&icde=53097928&ddparam=&ddvalue=&ddsub=&cr=1&csb=default&cs=ASC&pball=) from the NIH for the type 1 diabetes work. The work focuses on data generated from the [TEDDY](https://teddy.epi.usf.edu/) consortium, of which my PhD mentor, Dr. Jin-Xiong She, is a PI.

A manuscript detailing the polygenic risk of islet autoimmunity in newborns is in preparation. [News article feature](https://jagwire.augusta.edu/tag/paul-tran/)

See my blog post on the difference between a PRS for islet autoimmunity compared to type 1 diabetes and why it is more important to predict islet autoimmunity [here](https://www.pmhtran.com/blog/islet_autoimmunity/).

__Genetic Mapping of Type 1 Diabetes Associated Variants__

We a have identified potential causal variants and regulatory mechanism linking the CCR2 locus to type 1 diabetes. Our data suggest that certain variants increase CCR2 gene expression in monocytes and effector T cells leading to increase CCR2/CCL2 pathway activation and immune destruction of pancreatic islets.

[The 3p21.31 genetic locus promotes progression to type 1 diabetes through the CCR2/CCL2 pathway] (https://www.sciencedirect.com/science/article/pii/S2589909021000472)

A manuscript detailing the genetics of serum Il1-ra and its association with progression to T1D is in progress. [code](https://rpubs.com/ptran/866046)

__The Serum Anti-Carbohydrate Antibody (ACA) Repertoire in Type 1 Diabetes__
We profiled serum ACAs against 202 glycans from 22 functional clasess in 688 T1D and control subjects from 2 clinical cohorts (PAGODA and DAISY). The manuscript will be submitted soon.

[Code](https://github.com/pmtran5884/ACA_associated_with_T1D)



__Type 1 Diabetes Complications__

This project is part of the Phenome and Genome of Diabetic Autoimmunity ( [PAGODA](https://www.augusta.edu/research/studies/trial.php?study=611249)) study initiated by my mentor, Jin-Xiong She, PhD through funding from NIDDK and JDRF.

The first manuscript describes the role of the modifiable risk factors, blood pressure and glucose level, in developing microvascular complications of type 1 diabetes. Links: 
[Code](https://github.com/pmtran5884/T1D_Complications), 
[Website](https://ptran25.shinyapps.io/Diabetic_Peripheral_Neuropathy_Risk/?_ga=2.159360185.2040679849.1609524909-2098218154.1607979380), and
[Manuscript 1](https://www.mdpi.com/1660-4601/18/21/11094)

The second manuscript describes the role of inflammatory proteins in developing diabetic peripheral neuropathy.
[Manuscript 2](https://www.frontiersin.org/articles/10.3389/fimmu.2021.654233/full)
Code will be released soon.


